### DISCUSSION TITLE
Coc.nvim and coc.pyright require different node versions to work! What do you guys do for this?

### DISCUSSION THREAD

**Discussion by SubhamBurnwal** [neoclide/coc.nvim#2832] [Q&A]
I am on Ubuntu LTS 10.04, and I installed coc.nvim with vim-plug.
coc.nvim asks for v10.12 or something while the coc.pyright extension requires v12+
Whenever I install either of those, the previous one gets uninstalled I guess.. I know this is the case because the warning pops up whenever I open Vim or whenever I open a python file(which turns on coc.pyright)
What do I do??

**Comment by chemzqm** [comment]
Use node 12 or above
